### PR TITLE
Angular Tasks

### PR DESCRIPTION
**Pull request by ibratnawaz**

### CODE REVIEW

**Comment by ibratnawaz** [comment]
@suraj-pareek, Please review it. Thanks

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=827846
```
Comment (suraj-pareek) [inline .vscode/extensions.json]: we shouldn't push this file

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
```
Comment (suraj-pareek) [inline .vscode/launch.json]: we shouldn't push this file

File: .vscode/tasks.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{
+  // For more information, visit: https://go.microsoft.com/fwlink/?LinkId=733558
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "type": "npm",
+      "script": "start",
+      "isBackground": true,
+      "problemMatcher": {
+        "owner": "typescript",
```
Comment (suraj-pareek) [inline .vscode/tasks.json]: we shouldn't push this file

File: src/app/components/navbar/navbar.component.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  getActiveLink() {
+    this.router.events.subscribe((event: Event) => {
+      if (event instanceof NavigationEnd) {
+        this.currentActiveLink = event.url;
+      }
+    });
+  }
+
+  onNavigation(link: string) {
+    console.log('>>>', this.router);
```
Comment (suraj-pareek) [inline src/app/components/navbar/navbar.component.ts]: please remove console.log

File: src/app/modules/shared/components/users-list/users-list.component.spec.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should emit with the given id', () => {
+    spyOn(component.showUser, 'emit').and.callFake((id: string) =>
+      console.log('>> id is ' + id)
```
Comment (suraj-pareek) [inline src/app/modules/shared/components/users-list/users-list.component.spec.ts]: please remove console

**Review by suraj-pareek — APPROVED** [review]
Good Work!
Please just remove the console.log at couple of places.

**Pull request closed without merging**